virttest/utils_test/qemu/__init__.py

```diff
@@ -0,0 +1,686 @@
+"""
+High-level QEMU test utility functions.
+
+This module is meant to reduce code size by performing common test procedures.
+Generally, code here should look like test code.
+
+More specifically:
+    - Functions in this module should raise exceptions if things go wrong
+    - Functions in this module typically use functions and classes from
+      lower-level modules (e.g. utils_misc, qemu_vm, aexpect).
+    - Functions in this module should not be used by lower-level modules.
+    - Functions in this module should be used in the right context.
+      For example, a function should not be used where it may display
+      misleading or inaccurate info or debug messages.
+
+:copyright: 2008-2013 Red Hat Inc.
+"""
+
+import logging
+import os
+import re
+import time
+from functools import reduce
+
+from avocado.core import exceptions
+from avocado.utils import path as utils_path
+from avocado.utils import process
+
+from virttest import error_context
+from virttest import utils_misc
+from virttest import qemu_monitor
+from virttest.qemu_devices import qdevices
+from virttest.staging import utils_memory
+
+
+def guest_active(vm):
+    o = vm.monitor.info("status")
+    if isinstance(o, str):
+        return "status: running" in o
+    else:
```

Comment:
This is raw code in the framework, I have not changed. And I think it is ok, it judges the return message from hmp or qmp.